scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -619,10 +622,12 @@
         considerable->task-id (plumbing.core/map-from-keys (fn [_] (str (java.util.UUID/randomUUID))) considerable)
         guuid->considerable-cotask-ids (util/make-guuid->considerable-cotask-ids considerable->task-id)
         running-cotask-cache (atom (cache/fifo-cache-factory {} :threshold (max 1 (count considerable))))
+        job-uuid->reserved-host (or (:reservations @rebalancer-reservation-atom) {})
         ; Important that requests maintains the same order as considerable
         requests (mapv (fn [job]
                          (make-task-request db job :task-id (considerable->task-id job) :guuid->considerable-cotask-ids guuid->considerable-cotask-ids
-                                            :running-cotask-cache running-cotask-cache))
+                                            :running-cotask-cache running-cotask-cache
+                                            :job-uuid->reserved-host job-uuid->reserved-host))
```

Comment:
Nitpick: lexicographic ordering of optional arguments. Also, it would be nice if we can align them.

Reply:
Reordered. Not going to align them.

Reply:
thanks, I meant justify, not align :)